.gitignore

```diff
@@ -42,3 +47,15 @@
 /packages/updating-element/decorators.*
 /packages/updating-element/platform-support.*
 /packages/updating-element/updating-element.*
+
+# lit-ssr package
+/packages/lit-ssr/lib
```

Comment:
The `lit-` prefix is used in all packages that will be published, afaict.  Except `updating-element`, which seems fine since it's not Lit-specific.  What else is bothering you about the current naming?